packages/online-editor/src/common/i18n/locales/en.ts

```diff
@@ -145,9 +145,10 @@ export const en: OnlineI18n = {
           introduction: `In order to create your ${en_common.names.shortDevSandbox} instance:`,
           goToGetStartedPage: "Go to the Get Started page",
           followSteps: `Follow the steps to launch your instance. You will be asked to log in with your ${en_common.names.redHat} account.`,
-          informUsername: `Once your instance is up and running, inform the username of your ${en_common.names.redHat} account below.`,
-          inputReason: "This information is necessary for locating your namespaces.",
-          usernamePlaceholder: `The username of your ${en_common.names.redHat} account`,
+          informNamespace: `Once your instance is up and running, inform the namespace (project) in your cluster you want to have the Decision Model deployed to.`,
+          inputReason:
+            "This information is necessary for deploying youde Decision Model into the right project namespace.",
```

Comment:
```suggestion
            "This information is necessary for deploying your Decision Model into the right project namespace.",
```
There is a type here.

Reply:
thanks